server/src/modules/users/application/use-cases/create-user/__tests__/create-user-use-case.test.unit.ts

```diff
@@ -45,7 +50,10 @@ describe('CreateUserUseCase', () => {
     const createUserResult = await createUserUseCase.execute(createUserDTO)
 
     expect(createUserResult.isErr()).toBe(true)
-    const createUserErr = createUserResult as Err<CreateUserSuccess, UserValueObjectErrors.InvalidEmail>
+    const createUserErr = createUserResult as Err<
+      CreateUserSuccess,
+      UserValueObjectErrors.InvalidEmail
+    >
     expect(createUserErr.error instanceof UserValueObjectErrors.InvalidEmail).toBe(true)
```

Comment:
another (also-ugly, but typesafe) way to do it is

```suggestion
    if (!createUserResult.isErr()) throw new Error('this is impossible')
    expect(createUserResult.error instanceof UserValueObjectErrors.InvalidEmail).toBe(true)
```